scripts/index.ts

```diff
@@ -37,79 +38,83 @@ async function run(steps: string[]) {
         ]
       );
       continue;
-    }
-
-    switch (step) {
-      case 'check-install':
-        await checkInstall();
-        break;
-      case 'encrypt-service-account-credentials':
-        const keyAndIV = await encryptCredentials(`node-team-test-${keyID}.json`);
-        console.log([
-          `key: ${keyAndIV.key}`,
-          `iv: ${keyAndIV.iv}`
-        ].join('\n'));
-        break;
-      case 'decrypt-service-account-credentials':
-        const {
-          TRACE_SYSTEM_TEST_ENCRYPTED_CREDENTIALS_KEY: key,
-          TRACE_SYSTEM_TEST_ENCRYPTED_CREDENTIALS_IV: iv,
-        } = process.env;
-      
-        if (!key || !iv) {
-          console.log('> Environment insufficient to decrypt service account credentials');
+    } else if (step.startsWith('compile-')) {
+      const [_literalCompile, languageLevel, strict] = step.split('-');
```

Comment:
`_literalCompile` is not used, right?

```ts
const [, languageLevel, strict] = step.split('-');
```
?

Reply:
I wasn't aware of this syntax... added :)